### ISSUE TITLE
Contribuições no site do PyLadies DF

### ISSUE THREAD

**Issue by rougeth** [grupydf/grupy-df.gerenciamento#3]
Vamos organizar e retomar as contribuições no site do PyLadies DF?

**Comment by humrochagf** [comment]
sim, uma coisa, @kamillacrozara como é o esquema que você falou de hospedagem do pyladies mesmo?

**Comment by gilsondev** [comment]
Como vai ser feito o site? Podemos marcar um Sprint

**Comment by gilsondev** [comment]
Abri uma issue do segundo Sprint, e coloquei isso no tópico. É a issue #7
